connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Loggers.java

```diff
@@ -105,14 +108,10 @@ public synchronized LoggerLevel level(String logger) {
     public synchronized Map<String, LoggerLevel> allLevels() {
         Map<String, LoggerLevel> result = new TreeMap<>();
 
-        currentLoggers().stream()
-                .filter(logger -> !logger.getLevel().equals(Level.OFF))
-                .forEach(logger -> result.put(logger.getName(), loggerLevel(logger)));
-
-        org.apache.logging.log4j.Logger root = rootLogger();
-        if (!root.getLevel().equals(Level.OFF)) {
-            result.put(ROOT_LOGGER_NAME, loggerLevel(root));
-        }
+        currentLoggers()
```

Comment:
It seems the implementation of `currentLoggers` is not correct - it should return all registered logger rather than configured logger. For example: `context.getLoggerRegistry().getLoggers()`